### PR TITLE
Feature/pokemon container

### PR DESCRIPTION
**Pull request by MakarCodes**

### CODE REVIEW

File: src/containers/PokemonContainer/PokemonContainer.js [hunk 1]
```diff
@@ -46,4 +40,9 @@ class PokemonContainer extends Component {
     }
 }
 
-export default PokemonContainer;
\ No newline at end of file
+const mapDisptchToProps = dispatch => {
```
Comment (tkleszczewski) [inline src/containers/PokemonContainer/PokemonContainer.js:43]: a Typo, change to mapDispatchToProps

File: src/store/reducers/pokemonReducer.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    switch(action.type) {
+        case actionTypes.FETCHING_DATA_START:
+            return {
+                ...state,
+                loading: true
+            }
+        case actionTypes.FETCHING_DATA_SUCCESS:
+            return {
+                ...state,
+                pokemons: 'Chicken',
```
Comment (tkleszczewski) [inline src/store/reducers/pokemonReducer.js:19]: change to action.pokemons

File: src/store/reducers/pokemonReducer.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import * as actionTypes from '../actions/actionTypes'
+
+const initialState = {
+    pokemons: 'Pikatchu',
```
Comment (tkleszczewski) [inline src/store/reducers/pokemonReducer.js:4]: change to pokemons: [],

Comment (MakarCodes) [reply src/store/reducers/pokemonReducer.js]: fixed

File: src/store/reducers/pokemonReducer.js (hunk 2 above)
Comment (MakarCodes) [reply src/store/reducers/pokemonReducer.js]: done my friend

File: src/containers/PokemonContainer/PokemonContainer.js (hunk 1 above)
Comment (MakarCodes) [reply src/containers/PokemonContainer/PokemonContainer.js]: fixed

File: src/store/reducers/pokemonReducer.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    loading: false,
+    error: false
+}
+
+const reducer = (state = initialState, action) => {
+    switch(action.type) {
+        case actionTypes.START_FETCHING_DATA:
```
[the remark this answers is not in the document]
Comment (MakarCodes) [reply src/store/reducers/pokemonReducer.js]: done

**Pull request merged**